velox/docs/functions/presto/math.rst

```diff
@@ -233,3 +233,14 @@ Floating Point Functions
 .. function:: nan() -> double
 
     Returns the constant representing not-a-number.
+
+
+====================================
+Probability Functions
+====================================
+
+.. function:: beta_cdf(a, b, value) -> double
+
+    Compute the Beta cdf with given a, b parameters:  P(N < value; a, b).
```

Comment:
Let's turn "Beta cdf" into a link to https://en.wikipedia.org/wiki/Beta_distribution

Reply:
@mbasmanova Fixed it